terraformer_iam.tf

```diff
@@ -67,7 +67,7 @@ data "aws_iam_policy_document" "terraformer_assume_delegated_role_policy_doc" {
       aws_iam_role.terraformer_role.arn,
       module.read_terraform_state.role.arn,
       data.terraform_remote_state.master.outputs.organizationsreadonly_role.arn,
-      aws_iam_role.gucamole_parameterstorereadonly_role.arn,
+      aws_iam_role.guacamole_parameterstorereadonly_role.arn,
```

Comment:
If the templating changes work I'm not sure if permission to assume this role is still needed so just flagging.

Reply:
Please see commit d9b1e38.